packages/@okta/vuepress-site/guides/sign-into-spa/configure-the-sdk/index.md

```diff
@@ -5,7 +5,7 @@ title: Configure the SDK
 You'll need two values from the Okta Application and the Developer Console you worked with in <GuideLink link="../create-okta-application">Create an Okta Application</GuideLink>:
 
 * **Client ID** - Find it in the applications list or on the application's **General** tab.
-* **Okta domain** - Find it on the Developer Console dashboard in the upper-right corner. 
+* **Okta domain** - Find it on the Developer Console dashboard in the upper-right corner (shown as `Org URL:`).
```

Comment:
@susanharper-okta Worth clarifying on other guides you think?

Reply:
Yes, worth clarifying - but with the below edit. I will update the other guides with this change.